winui/Cartesian-Charts/Legend.md

```diff
@@ -378,103 +378,18 @@ By clicking on disabled legend item, we can view the associated `series`,
 
 ![ToggleSeriesVisibility support for legend in WinUI Chart](Legend_images/WinUI_chart_legend_toggleSeriesVisibility1.png)
 
-## Position
+## Placement
 
-The legends can be placed either inside or outside of the chart area (plotting area). By default, it will be displayed outside and positioned at top (using [Position](https://help.syncfusion.com/cr/winui/Syncfusion.UI.Xaml.Charts.ChartLegend.html#Syncfusion_UI_Xaml_Charts_ChartLegend_Position)) of the chart area.
+Legends can be docked left, right, and top or bottom around the chart area using [Placement]() property. By default, the chart legend is docked at the top of the chart as mentioned earlier.
```

Comment:
Legends can be docked left, right, and top or bottom around the chart area using the [Placement]() property. By default, the chart legend is docked at the top of the chart as mentioned earlier.